src/components/ContributionAssistantPriceFormCard.vue

```diff
@@ -16,12 +24,20 @@
     />
     <v-card-text>
       <ProductInputRow :productForm="productPriceForm" :disableInitWhenSwitchingModes="true" @filled="productFormFilled = $event" />
+      <v-alert
```

Comment:
last bit missing, is to set the mode as barcode if a product_code is detected ?? or is there many false-positives ?

Reply:
Sorry, didn't understand this comment.

But maybe this context helps: gemini always tries its best to but a value to all fields. So its reply almost always contains a barcode, even when no barcodes were in the image (it likes to put the price in the barcode field for these cases)

Related fun fact: I've noticed it also does something similar with origin countries that are not in its enum set, replacing it with countries that are close geographically or culturally. For example, if its set includes CHINA and BRAZIL, and the read origin is CHILE, it tends to reply with BRAZIL, instead of the closer word CHINA, which is surprising (and pretty cool).

Reply:
So the suggestion is to have "barcode" selected by default instead of "category" ?

I wonder if users would also get lost for crops without barcodes, with gemini putting a random number as barcode, and users not knowing that the category is hidden behind the other mode.

Reply:
I need to take a closer look in the gemini response. Or maybe couple gemini with a classifier ? That would tell us if the image is a barcode or a category ??

Reply:
There's probably a way of modifying the prompt, asking specifically for EAN13 barcodes or empty strings. It might even lead to better parsing results !

Reply:
See #1042